### DISCUSSION TITLE
Edited: uid empty error at at FirebaseTokenGenerator - createCustomToken

### DISCUSSION THREAD

**Discussion by thanhtutzaw** [gladly-team/next-firebase-auth#581] [Q&A]
When I fetch data with SSR , I got this error message and  sometimes I don't get this error message . What is the problem . Cleaning browser storage also didn't work . **But I can Tell I added firebase config by my self for db in SSR**.

![Screenshot (86)](https://user-images.githubusercontent.com/71011043/201507697-b2b568b0-f58a-4c53-a028-ed2e2f2138d5.png)

### Firebase.js (getting db from here )

![Screenshot (87)](https://user-images.githubusercontent.com/71011043/201515108-14c8a20d-83a6-446d-8f81-f2ccba25f4f6.png)

### SSR in indexjs

![Screenshot (88)](https://user-images.githubusercontent.com/71011043/201515457-cb8728e4-9843-4506-85c1-5666c4fd67da.png)

### Update
I noticed today firestore read increased from 16 to 164  ( get all (2data) from one user ) and (get single data with 
dynamic-route)

![Screenshot (90)](https://user-images.githubusercontent.com/71011043/201517880-34e4b490-d39f-4b82-ad6d-cba253e45432.png)

**Comment by kmjennison** [comment]
Possibly related:
https://github.com/gladly-team/next-firebase-auth/issues/411
https://github.com/gladly-team/next-firebase-auth/issues/531
